### PR TITLE
igl conda package

### PR DESCRIPTION
**Pull request by teseoch**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/igl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    - scipy
+
+test:
+  imports:
+    - igl
+
+about:
+  home: https://libigl.github.io
+
+  license: GPL
```
Comment (ocefpaf) [inline recipes/igl/meta.yaml]: What version of GPL? (Please use the SPDX string for it.)

File: recipes/igl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  number: 0
+  skip: True  # [win]
+  script:
+    - "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - git
```
Comment (ocefpaf) [inline recipes/igl/meta.yaml]: Is `git` needed at build time? If so can it be at `host`?

File: recipes/igl/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    - cmake
+
+  host:
+    - python
+    - pip
+    - setuptools
```
Comment (ocefpaf) [inline recipes/igl/meta.yaml]: With `pip` you can remove `setuptools` here.

File: recipes/igl/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    - libblas
+    - libcblas
+    - numpy >=1.9.1
```
Comment (ocefpaf) [inline recipes/igl/meta.yaml]: Use just `numpy` and let the global pinning take care of the minimum version.
With the `>=` here you are not building with `1.9`, you'll probably get `1.16` and then people with `<1.16` will have trouble using this package.

File: recipes/igl/meta.yaml (hunk 2 above)
Comment (teseoch) [reply recipes/igl/meta.yaml]: no, thanks

File: recipes/igl/meta.yaml (hunk 1 above)
Comment (teseoch) [reply recipes/igl/meta.yaml]: ok fixed

File: recipes/igl/meta.yaml (hunk 3 above)
Comment (teseoch) [reply recipes/igl/meta.yaml]: sure, done

File: recipes/igl/meta.yaml (hunk 4 above)
Comment (teseoch) [reply recipes/igl/meta.yaml]: ok

File: recipes/igl/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  imports:
+    - igl
+
+about:
+  home: https://libigl.github.io
+
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE.GPL
+
```
Comment (ocefpaf) [inline recipes/igl/meta.yaml]: No need for the extra spaces in the `about` section.

File: recipes/igl/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    - "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+
+  host:
+    - git
```
Comment (ocefpaf) [inline recipes/igl/meta.yaml]: I'm curious, why do you need `git` for the installation process?

Comment (teseoch) [reply recipes/igl/meta.yaml]: mhmh you are right

File: recipes/igl/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+  description: |
+    libigl is a simple python and C++ geometry processing library. We have a wide functionality including construction of sparse discrete differential geometry operators and finite-elements matrices such as the cotangent Laplacian and diagonalized mass matrix, and simple facet and edge-based topology data structures.
+  doc_url: https://libigl.github.io/tutorial/
+  dev_url: https://github.com/libigl/libigl/
+extra:
+  recipe-maintainers:
+    - teseoch
+    - fwilliams
+    - skoch9
+    - danielepanozzo
```
Comment (ocefpaf) [inline recipes/igl/meta.yaml]: @fwilliams, @skoch9, and @danielepanozzo are you OK to be maintainers here?

File: recipes/igl/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+    - igl
+
+about:
+  home: https://libigl.github.io
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE.GPL
+  summary: 'Simple python geometry processing library'
+  description: |
+    libigl is a simple python and C++ geometry processing library. We have a wide functionality including construction of sparse discrete differential geometry operators and finite-elements matrices such as the cotangent Laplacian and diagonalized mass matrix, and simple facet and edge-based topology data structures.
```
Comment (ocefpaf) [inline recipes/igl/meta.yaml]: You can break this into a few lines to make it more readable.

File: recipes/igl/meta.yaml (hunk 7 above)
Comment (danielepanozzo) [reply recipes/igl/meta.yaml]: yes

Comment (skoch9) [reply recipes/igl/meta.yaml]: yes

Comment (fwilliams) [reply recipes/igl/meta.yaml]: Yes

File: recipes/igl/meta.yaml (hunk 8 above)
Comment (teseoch) [reply recipes/igl/meta.yaml]: ok done

**Comment by teseoch** [comment]
is it ok? can it be merged?

**Comment by ocefpaf** [comment]
Something is not OK with azure :-/ let's try to close and re-open to see if we can toggle the CIs.

**Comment by teseoch** [comment]
> Something is not OK with azure :-/ let's try to close and re-open to see if we can toggle the CIs.

seems to do the trick

**Pull request merged**